trtools/dumpSTR/tests/test_dumpSTR.py

```diff
@@ -6,6 +6,7 @@
 import vcf
 
 from ..dumpSTR import *
+from trtools.testsupport.utils import assert_same_vcf, assert_same_file
 
 
```

Comment:
Is this file meant to be deleted? It's in dumpSTR

Reply:
The entire file isn't deleted, just those methods, which have been moved to trtools/testsupport/utils.py